tensorflow/stream_executor/dso_loader.cc

```diff
@@ -78,10 +78,20 @@ string GetCudnnVersion() { return TF_CUDNN_VERSION; }
                   GetCudaDriverLibraryPath()),
       dso_handle);
 #else
-  return GetDsoHandle(
+  port::Status status = GetDsoHandle(
       FindDsoPath(port::Env::Default()->FormatLibraryFileName("cuda", "1"),
                   GetCudaDriverLibraryPath()),
       dso_handle);
+#if defined(__APPLE__)
+  // On Mac OS X, CUDA sometimes installs libcuda.dylib instead of
```

Comment:
CUDA seems to have the "1" on Linux systems. I don't know if the difference between Mac and Linux is intended. But in case it's not intended on Mac and NVIDIA fixes later, we won't have to make changes because of the code here.